### ISSUE TITLE
bug: Typescript - Property 'session' does not exist on type 'Context'

### ISSUE THREAD

**Issue by foochris** [grammyjs/grammY#385]
Following the [Free Storage](https://grammy.dev/plugins/session.html#free-storage) example and using a Typescript file, I'm encountering this fatal error:

```
TSError: ⨯ Unable to compile TypeScript:
bot.ts:60:19 - error TS2339: Property 'session' does not exist on type 'Context'.
60   console.log(ctx.session.count)
```

**Expected**
* console output of `0`

**Actual**
* fatal error that causes the node app to crash

**Workaround**
```
// eslint-disable-next-line
// @ts-ignore
console.log(ctx.session.count)
```

**Environment** 
GrammY: 1.15.1
Node: 18.14.1

**Comment by KnorpelSenf** [comment]
The code compiles just fine for me. Can you share the exact code you're using, as well as your `tsconfig.json` and TypeScript version?

**Comment by KnorpelSenf** [comment]
> @KnorpelSenf Thanks for the prompt reply. I believe the issue is:
> 
> ```
> bot.on('message', async (ctx:Context) => {
> ```

Yes, you are right. You should never annotate a variable if its can be inferred correctly.

If you just do `bot.on('message', async (ctx) => {` then it will work correctly.

Please close this issue if you can confirm that this solves the problem.
